ApeWorX/ape
# feat: show gas reports from Ethereum transaction receipts

### What I did

First step to showing gas reports in tests.
Adds a way to show one on a receipt.

### How I did it

* Share code with tracing stuff.

### How to verify it

Get a txn receipt from somewheres, like gran a txn hash from Etherscan and get the receipt via `provider.get_receipt()` and then call `show_gas_usage()`.

See my trace project as an example and could even use that, that is what I do: https://github.com/unparalleled-js/ape-trace-project#gas

### Checklist
